### DISCUSSION TITLE
Traces deleted in past

### DISCUSSION THREAD

**Discussion by andrewgkew** [apache/skywalking#7759] [Q&A] [answered]
Hi
I have an instance of Skywalking running and I have setup some data based on scenarios for a presentation I am doing next week.
When I go to the topology view and I go back in time everything looks fine, but then when I try and view the traces around that time I am getting no data.

All the k8s metrics though seem to be there.

Is the tracing information deleted by any chance?

**Comment by wu-sheng** [comment, marked as answer]
I think you are asking for this, https://skywalking.apache.org/docs/main/latest/en/setup/backend/ttl/

**Reply by andrewgkew** [reply]
Thats exactly what I am looking for thanks @wu-sheng 

And are these env vars that I pass into the agent or the skywalking install (via helm)?

**Reply by wu-sheng** [reply]
Those belong to backend server. So `via helm`, yes.

**Reply by andrewgkew** [reply]
Yup just logged onto the OAP server and found the property in the application.yml file under core.default

So I assume I can do the following

```
--set oap.env.SW_CORE_RECORD_DATA_TTL=10
```

**Reply by wu-sheng** [reply]
Yes, you can.
